frontend/flaskserver.py

```diff
@@ -80,10 +80,22 @@ def get_mime_string(path):
     secure = True
 
 # SSL context
-context = ('selfsigned.crt', 'decserver.key')
+context = (exec_dir+'/selfsigned.crt', exec_dir+'/decserver.key')
 
 # ROUTES ###############################################################
 
+@app.after_request
+def add_header(r):
+    """
+    Add headers to both force latest IE rendering engine or Chrome Frame,
+    and also to cache the rendered page for 10 minutes.
+    """
+    r.headers["Cache-Control"] = "no-cache, no-store, must-revalidate"
+    r.headers["Pragma"] = "no-cache"
+    r.headers["Expires"] = "0"
```

Comment:
I don't think it even matters with "no-cache" set.
The reason why I disabled cache is to
* remove some potential problems when debugging stuff and the client doesn't refresh the cache
* firefox doesn't like angularjs applications with cache on, loading is really slow